### PR TITLE
[Menubar] New primitive

### PR DESCRIPTION
**Pull request by andy-hook**
Base branch for `Menubar` implementation.

- [x] https://github.com/radix-ui/primitives/pull/1847
- [x] https://github.com/radix-ui/primitives/pull/1852
- [x] https://github.com/radix-ui/primitives/pull/1881
- [x] https://github.com/radix-ui/primitives/pull/1883

### CODE REVIEW

**Comment by andy-hook** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1846** <a href="https://app.graphite.dev/github/pr/radix-ui/primitives/1846" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/radix-ui/primitives/1846?utm_source=stack-comment).

File: .yarn/versions/a07c2b66.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  "@radix-ui/react-menu": patch
+  "@radix-ui/react-radio-group": patch
+  "@radix-ui/react-roving-focus": patch
+  "@radix-ui/react-tabs": patch
+  "@radix-ui/react-toggle-group": patch
+  "@radix-ui/react-toolbar": patch
+
+declined:
+  - primitives
+  - "@radix-ui/react-menubar"
```
Comment (andy-hook) [inline .yarn/versions/a07c2b66.yml:13]: Do I have these right?

Comment (benoitgrelard) [reply .yarn/versions/a07c2b66.yml:13]: `"@radix-ui/react-menubar"` should be major right?

Comment (andy-hook) [reply .yarn/versions/a07c2b66.yml:13]: Good question, should the first release be a straight major?

**Review by benoitgrelard — APPROVED** [review]

**Pull request merged**